src/components/AddressSelect/AddressesSelect.tsx

```diff
@@ -108,6 +124,10 @@ function AddressesSelect({ setUsersAddresses, isDisabled, maxAddressesAmount, al
         validate: (address) => !isDuplicate(address),
         errorKey: 'component.addresses_select.duplicated_error',
       },
+      {
+        validate: (address) => (loggedUserIsInvalid ? !isAuthor(address) : true),
+        errorKey: loggedUserInvalidKey || 'component.addresses_select.logged_user_invalid',
```

Comment:
We could but we'd loose clarity in the message (the coauthors select talks about the proposal author for example: "This wallet address cannot be added since it belongs to the proposal's author.")